stack/indexer/deb/build_package.sh

```diff
@@ -50,6 +50,15 @@ build_deb() {
             base_cmd+="--reference ${reference}"
         fi
         ../base/generate_base.sh -s ${outdir} -r ${revision} ${base_cmd}
+    else
+        version=$(cat ${current_path}/../../../VERSION)
```

Comment:
builder.sh uses "reference"'s branch VERSION file rather than the local one. Did you consider this?

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/2462/commits/526b7d8ea905b0a6ece4df4f1d7fafa2ea6ae3f9